lib/AST/Expr.cpp

```diff
@@ -1812,26 +1807,25 @@ ConditionalCheckedCastExpr *
 ConditionalCheckedCastExpr::createImplicit(ASTContext &ctx, Expr *sub,
```

Comment:
Not your bug, but we need to eliminate this constructor entirely. Its sole usage makes no sense...